### DISCUSSION TITLE
Concretely defining the criteria

### DISCUSSION THREAD

**Discussion by nguyenhoangan-monkey** [LAION-AI/Open-Assistant#3455] [Ideas]
Currently there's a lot of confusion about our labeling criteria, so I propose modifying the labels of the likert scale a bit in this image:
<img width="742" alt="Screenshot 2023-03-10 at 14 32 20" src="https://user-images.githubusercontent.com/121211419/224253601-ce1f1d69-a36b-4273-9853-3d96549a4fb3.png">

**Comment by andreaskoepf** [comment]
The extra column with quick help makes sense. But we don't want words like bs in our label descriptions. The help-text should be written by a native speaker.

The robotic - human label would be a new label, i.e. incompatible to the existing quality label. Wikipedia as value '3' for useful seems to me a bit low. Is 'useful' really better than the current 'helpful'?
